satellite/contact/nodesservice.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package contact
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/pb"
+)
+
+// NodesServiceEndpoint implements the NodesServer Interface
+type NodesServiceEndpoint struct {
+	log *zap.Logger
+}
+
+// NewNodesServiceEndpoint returns a new nodes service endpoint
+func NewNodesServiceEndpoint(log *zap.Logger) *NodesServiceEndpoint {
```

Comment:
i feel like usually something is a service or it is an endpoint, why is this both?

Reply:
Yeah that is weird I agree. It's because it's implementing the pb.NodesServer interface which is an rpc service. And it seemed like it wasn't actually a server... We can definitely change the name... do you have any ideas?

Reply:
How about `KademliaEndpoint`?

Reply:
yeah! or `KadCompadEndpoint`?

Reply:
Either way, it's only going to exist for a release or two, so it's probably not a big deal

Reply:
how about just kademliaendpoint then

Reply:
all those seem good